### ISSUE TITLE
Showing error message when not in 100% in viewport

### ISSUE THREAD

**Issue by mhretab** [twokul/ember-lazy-image#29]
Hi,
I am using ember 1.11.2 with cli 0.2.3 and the component is showing an error message  when the image is not fully inside a view port. when I try to override  ember-in-viewport options the component crashes. is this compatibility issue?

**Comment by twokul** [comment]
@mhretab probably. do you mind putting a quick example together to help me track the issue down?

**Comment by mhretab** [comment]
@twokul here is what I did. Installed ember-lazy-image, and tried to use it with height like this:
```handlebars
{{lazy-image url=src width=424 height=height class="post-image"}}
```
That one worked as long as the image is 100% in the viewport, otherwise it displayed error message instead of the loading bar.
Then I tried to extend the component as per in the documentation so that I can override ember-in-viewport option. However when I tried that the component just crashed. I have seen that ember-in-viewport had an [issue (#19)](https://github.com/dockyard/ember-in-viewport/issues/19) with options and has been fixed with the latest release. I have tried upgrading ember-in-viewport to extend the component but still doesn't work.

**Comment by hbaughman** [comment]
It seems that this happens because the [error handler](https://github.com/twokul/ember-lazy-image/blob/master/addon/mixins/image-load.js#L30) is triggered when setting the (offscreen) image source to `//:0`.

I was able to fix this problem by changing `//:0` to `assets/spinner.gif` on [lazy-image.js:14](https://github.com/twokul/ember-lazy-image/blob/master/addon/mixins/lazy-image.js#L14) and :32, but this feels kludgy. You could try using a tiny base64 encoded gif to [make the `img` valid](http://stackoverflow.com/questions/5775469/whats-the-valid-way-to-include-an-image-with-no-src), but my cursory attempts at that failed. In any case that solution requires changing your `ENV.contentSecurityPolicy.img-src` policy in `config/environment.js` which may be undesirable.


I'm using jQuery 1.11.3 in Chrome 42.0.2311.135 and Firefox 37.0.2. I'm pretty sure this error doesn't have anything to do with ember, but I'm using Ember 1.11.1 and ember-cli 0.2.3.

You can test that `src='//:0'` raises an error thusly:

```html
<!DOCTYPE html>
<html>
<head>
  <title>Trigger errors by inserting invalid images</title>
</head>

<script type="text/javascript" src="/bower_components/jquery/dist/jquery.min.js"></script>
<script type="text/javascript">
$(document).on('ready', function() {
  $("img").on('error', function() {
    alert("<img src='//:0'> triggers error");
  });
});
</script>
<body>
  <img src="//:0">
</body>
</html>
```

**Comment by hbaughman** [comment]
Looks like my hack causes images that fail to load to display a broken image instead of the errorText.

**Comment by twokul** [comment]
@hbaughman @mhretab looking at this right now

**Comment by twokul** [comment]
published as `0.0.12`
